.taskcluster.yml

```diff
@@ -12,82 +12,99 @@ policy:
           head_ref:
               $switch:
                   'tasks_for[:19] == "github-pull-request"': ${event.pull_request.head.ref}
                   'tasks_for == "github-push"': ${event.ref}
-                  'tasks_for in ["cron", "action"]': '${push.branch}'
+                  'tasks_for == "github-release"': ${event.release.tag_name}
+                  'tasks_for in ["cron", "action", "pr-action"]': '${push.branch}'
           base_sha:
               $switch:
                   'tasks_for == "github-push"': '${event.before}'
+                  'tasks_for == "github-release"': '${event.release.target_commitish}'
                   'tasks_for[:19] == "github-pull-request"': '${event.pull_request.base.sha}'
-                  'tasks_for in ["cron", "action"]': '${push.revision}'
+                  'tasks_for in ["cron", "action", "pr-action"]': '${push.revision}'
           head_sha:
               $switch:
                   'tasks_for == "github-push"': '${event.after}'
+                  'tasks_for == "github-release"': '${event.release.tag_name}'
                   'tasks_for[:19] == "github-pull-request"': '${event.pull_request.head.sha}'
-                  'tasks_for in ["cron", "action"]': '${push.revision}'
+                  'tasks_for in ["cron", "action", "pr-action"]': '${push.revision}'
           ownTaskId:
               $switch:
                   '"github" in tasks_for': {$eval: as_slugid("decision_task")}
-                  'tasks_for in ["cron", "action"]': '${ownTaskId}'
+                  'tasks_for in ["cron", "action", "pr-action"]': '${ownTaskId}'
           pullRequestAction:
               $switch:
                   'tasks_for[:19] == "github-pull-request"': ${event.action}
                   $default: 'UNDEFINED'
+          releaseAction:
+              $if: 'tasks_for == "github-release"'
+              then: ${event.action}
+              else: 'UNDEFINED'
           isPullRequest:
               $eval: 'tasks_for[:19] == "github-pull-request"'
       in:
           $let:
               short_base_ref:
-                  $if: 'base_ref[:11] == "refs/heads/"'
-                  then: {$eval: 'base_ref[11:]'}
-                  else: ${base_ref}
+                  $switch:
+                      'base_ref[:10] == "refs/tags/"': '${base_ref[10:]}'
+                      'base_ref[:11] == "refs/heads/"': '${base_ref[11:]}'
+                      $default: '${base_ref}'
               short_head_ref:
-                  $if: 'head_ref[:11] == "refs/heads/"'
-                  then: {$eval: 'head_ref[11:]'}
-                  else: ${head_ref}
+                  $switch:
+                      'head_ref[:10] == "refs/tags/"': '${head_ref[10:]}'
+                      'head_ref[:11] == "refs/heads/"': '${head_ref[11:]}'
+                      $default: '${head_ref}'
+              level:
+                  $if: 'isPullRequest || tasks_for == "pr-action"'
+                  then: "1"
+                  else: "1"
```

Comment:
🤔 This could be just a comment suggesting in most cases we wouldn't want level 1 outside PRs?
Or at least add a comment explaining why we're setting level to 1 in both cases